src/storage/v2/storage.cpp

```diff
@@ -1144,6 +1144,16 @@ void Storage::Accessor::FinalizeTransaction() {
   }
 }
 
+uint64_t Storage::Accessor::GetTransactionId() const { return transaction_.transaction_id; }
+
+bool Storage::Accessor::IsTransactionActive() const { return is_transaction_active_; }
+
+std::atomic<bool> *Storage::Accessor::TransactionAbortedByUser() { return &transaction_.aborted_by_user; }
```

Comment:
When function returns bool, usually we are naming them with Is...

In this case you can change naming to IsTransactionAbortedByUser